packages/bot/src/scratch/blocks/Text/text_prompt_ext.js

```diff
@@ -3,15 +3,13 @@ import { translate } from '../../../utils/lang/i18n';
 Blockly.Blocks.text_prompt_ext = {
     init() {
         this.jsonInit(this.definition());
+        this.setOutputShape(Blockly.OUTPUT_SHAPE_ROUND);
```

Comment:
This can be set in the block's definition (see next comment).

```suggestion
```

Reply:
done